src/js/tech/tech.js

```diff
@@ -290,6 +298,30 @@ class Tech extends Component {
     if (this.manualTimeUpdates) { this.trigger({ type: 'timeupdate', target: this, manuallyTriggered: true }); }
   }
 
+
+  /**
+   * Initialize videotrack listeners
+   *
+   * @method initVideoTrackListeners
+   */
+  initVideoTrackListeners() {
+    let videoTrackListChanges = () => {
+      this.trigger('videotrackchange');
+    };
+
+    let tracks = this.videoTracks();
+
+    if (!tracks) return;
```

Comment:
If `videoTracks` always returns a track list, we can get rid of this short-circuiting.